### PR TITLE
remove base exception column + minor fixes

### LINKED ISSUE #4554: Horizontal scroll on narrow screens cuts off table content
See image below:

image

Seen on https://docs.microsoft.com/en-us/dotnet/standard/exceptions/ 

On a narrow screen, the main content panel has a slight horizontal scroll bar, but it doesn't scroll far enough to view all content, the ends of lines inside the table are still cut off.

My screen is 1280x1024, this was in Chrome 64.0.3282.186.

### LINKED ISSUE #5419: Table missing
After this text there's no table: "The following table lists the valid names for keyword values within the ConnectionString."
The table exists in the old article: https://msdn.microsoft.com/en-us/library/system.data.sqlclient.sqlconnection.connectionstring(v=vs.110).aspx

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8dbe2d02-339d-30f3-c267-cd5962abc4e4
* Version Independent ID: aa71bfe8-76a9-69d9-18d4-b5c22f7f8843
* Content: [SqlConnection.ConnectionString Property (System.Data.SqlClient)](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnection.connectionstring?view=netframework-4.7.1#System_Data_SqlClient_SqlConnection_ConnectionString)
* Content Source: [xml/System.Data.SqlClient/SqlConnection.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Data.SqlClient/SqlConnection.xml)
* Service: **.net**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #4554

### CODE REVIEW

File: docs/standard/exceptions/index.md [hunk 1]
```diff
@@ -30,38 +30,37 @@ Traditionally, a language's error-handling model relied on either the language's
...
-
-To learn more about how exceptions work in .NET, see [What Every Dev needs to Know About Exceptions in the Runtime](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/exceptions.md).
+| Exception type | Description | Example |
+| -------------- | ----------- | ------- |
+| <xref:System.Exception> | Base class for all exceptions. | None (use a derived class of this exception). |
+| <xref:System.IndexOutOfRangeException> | Thrown by the runtime only when an array is indexed improperly. | Indexing an array outside its valid range: <br /> `arr[arr.Length+1]` |
+| <xref:System.NullReferenceException> | Thrown by the runtime only when a null object is referenced. | `object o = null;` <br /> `o.ToString();` |
+| <xref:System.InvalidOperationException> | Thrown by methods when in an invalid state. | Calling `Enumerator.GetNext()` after removing an Item from the underlying collection. |
+| <xref:System.ArgumentException> | Base class for all argument exceptions. | None (use a derived class of this exception). |
+| <xref:System.ArgumentNullException> | Thrown by methods that do not allow an argument to be null. | `String s = null;` <br /> `"Calculate".IndexOf (s);` |
```
Comment (rpetrusha) [inline docs/standard/exceptions/index.md]: nit: remove the space between IndexOf and (s).

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. Just left a single nit. You can merge when ready.

**Comment by pkulikov** [comment]
@mairaw haven't you made a typo in the issue number this PR fixes?

**Comment by mairaw** [comment]
Yes I did @pkulikov! Thanks for calling that out.

File: docs/standard/exceptions/index.md [hunk 2]
```diff
@@ -30,38 +30,37 @@ Traditionally, a language's error-handling model relied on either the language's
...
-* [Using User-Filtered Exception Handlers](using-user-filtered-exception-handlers.md)
-* [How to: Use Finally Blocks](how-to-use-finally-blocks.md)
-* [Handling COM Interop Exceptions](handling-com-interop-exceptions.md)
-* [Best Practices for Exceptions](best-practices-for-exceptions.md)
-
-To learn more about how exceptions work in .NET, see [What Every Dev needs to Know About Exceptions in the Runtime](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/exceptions.md).
+| Exception type | Description | Example |
+| -------------- | ----------- | ------- |
+| <xref:System.Exception> | Base class for all exceptions. | None (use a derived class of this exception). |
+| <xref:System.IndexOutOfRangeException> | Thrown by the runtime only when an array is indexed improperly. | Indexing an array outside its valid range: <br /> `arr[arr.Length+1]` |
```
Comment (pkulikov) [inline docs/standard/exceptions/index.md]: nit: current code example is correct (example of an exceptional case), but what about `arr[arr.Length]` to clearly show that indexing is zero-based?

File: docs/standard/exceptions/index.md [hunk 3]
```diff
@@ -30,38 +30,37 @@ Traditionally, a language's error-handling model relied on either the language's
...
+| <xref:System.NullReferenceException> | Thrown by the runtime only when a null object is referenced. | `object o = null;` <br /> `o.ToString();` |
+| <xref:System.InvalidOperationException> | Thrown by methods when in an invalid state. | Calling `Enumerator.GetNext()` after removing an Item from the underlying collection. |
```
Comment (pkulikov) [inline docs/standard/exceptions/index.md]: It should be `Enumerator.MoveNext()`

**Review by pkulikov — COMMENTED** [review]
Left two nits related to the code examples.

File: docs/standard/exceptions/index.md (hunk 2 above)
Comment (mairaw) [reply docs/standard/exceptions/index.md]: I think we could keep the very obvious example. @rpetrusha @BillWagner thoughts?

Comment (rpetrusha) [reply docs/standard/exceptions/index.md]: I agree,, @mairaw. There's still occasional confusion about one-based and zero-based indexing, so this makes it obvious that the index is out of bounds without requiring explanation.

**Review by rpetrusha — APPROVED** [review]
As we discussed, you can merge this once you're satisfied with the appearance of multi-line code fragments, @mairaw.

**Pull request merged**